provider/repodata/data/json/fallback/likelysubtags@1/und.json

```diff
@@ -1,18 +1,59 @@
 {
   "l2s": {
```

Comment:
Thought/Issue, not your fault: we should filter this list so that it doesn't grow so much on this update. I worked out a solution for the main likelysubtags but not this fallback side version. Actually I would like to deprecate this key entirely and we can do that now.